demo/app.component.ts

```diff
@@ -107,6 +108,25 @@ export class AppComponent implements OnInit {
     this.form2 = this.formBuilder.group({ 'range': [ [ 2, 8 ] ] });
   }
 
+  public onSliderInit(slider: any) {
+    this.sliderObject  = slider;
+    console.debug('Slider Object Instance Initiated and available for use in your component');
+    console.debug(this.sliderObject);
+  }
+
+  public resetSlider() {
+    this.sliderObject.reset();
+  }
+
+  public togglePips() {
```

Comment:
Just do `(init)="doSomethingWithSlider($event)"` as defined [here](https://github.com/emonidi/ng2-nouislider/blob/5b60b993f43f2d5ab30180d111e924dbe587e129/src/nouislider.ts#L172 ). Also you can use `#mySliderElement` and call it in other element in template with `mySliderElement.slider.someNativeSliderMethod()`